subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/StubTransport.kt

```diff
@@ -0,0 +1,27 @@
+package com.avito.android.test.report.transport
+
+import com.avito.android.test.report.ReportState
+import com.avito.android.test.report.model.TestMetadata
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
+
+internal object StubTransport : Transport {
```

Comment:
NoOp

Reply:
If I'm not mistaken we agreed to use stubs everywhere:
https://github.com/avito-tech/avito-android/discussions/679

Reply:
Everywhere in tests I thought

Reply:
Maybe. 
`-test/test-report/..` confused me. I thought that this is a test fixture.